metrics.go

```diff
@@ -7,26 +7,29 @@ import (
 	"io"
 	"os"
 	"strconv"
+	"strings"
 	"syscall"
 
 	"github.com/pkg/errors"
 )
 
-type logLine struct {
-	Status    string `json:"status"`
-	Bytes     string `json:"bytes"`
-	BytesSent string `json:"bytes_sent"`
-	Hostname  string `json:"hostname"`
-}
-
 var (
 	filepath string
 )
 
-func (l logLine) getBytes() int {
-	bytes, _ := strconv.Atoi(l.Bytes)
+func sanitizeValue(label string, value string) string {
+	switch label {
+	case "content_type":
+		return strings.TrimSpace(strings.Split(value, ";")[0])
+	default:
+		return value
+	}
+}
```

Comment:
impose a max length on everything regardless of label?

Reply:
maybe even trim all leading and trailing whitespace?

Reply:
Any idea on a sane limit. Prometheus doesn't have one, and seems against adding a hard limit at their end: https://github.com/prometheus/prometheus/issues/1235

Reply:
256 for want of a better number?

Reply:
256 is probably enough for anybody. I'd go lower if we have data suggesting we can - its easier to allow longer labels in the future than it is to add further constraints.